curvefs/src/client/common/common.h

```diff
@@ -68,18 +68,20 @@ std::ostream &operator<<(std::ostream &os, MetaServerOpType optype);
 const uint32_t MAX_XATTR_NAME_LENGTH = 255;
 const uint32_t MAX_XATTR_VALUE_LENGTH = 64 * 1024;
 
-const char kCurveFsWarmupXAttr[] = "curvefs.warmup.op";
+const char kCurveFsWarmupListXAttr[] = "curvefs.warmup.list";
+const char kCurveFsWarmupQueryXAttr[] = "curvefs.warmup.query";
+const char kCurveFsWarmupAddXAttr[] = "curvefs.warmup.add";
+const char kCurveFsWarmupCancelXAttr[] = "curvefs.warmup.cancel";
```

Comment:
The initial design includes unnecessary aspects.

In the original process, we rely on `IsWarmupReq` to identify whether an operation is a warm-up request.
Subsequently, we employ `GetWarmupOpType` to determine the exact operation (To accommodate this, we need to add another string like `add`, `query`, `list`, and `cancel` to the parameter list).

I think the original design doesn't offer significant advantages.
Instead of using both curvefs.warmup.op and an additional string for operation type, it seems clearer to utilize `curvefs.warmup.[operation]`, such as `curvefs.warmup.list`.

Reply:
@Cyber-SiKu 
Upon reevaluation, another reason for this change is to distinguish between list and query operations. Currently, both operations use `getXttr`, which prevents the inclusion of `opType` in the `xttr` values, as evidenced by the direct use of `curvefs.warmup.query` commands.

To align the list/query operations with the warmup add process, another approach would be to use `setXttr` in the tool instead of `getXttr`, allowing for the inclusion of `opType` in the FUSE operation.

Reply:
Had a discussion with @Cyber-SiKu offline
To maintain backward compatibility while improving the readability, we decided:
* warmup add remains the conventional way, so does warmup cancel 
i.e., `curve.warmup.op` + add/cancel (FuseOpSetXattr)
* warmup query remains the conventional way
i.e., `curve.warmup.op` (FuseOpGetXattr)
* warmup list uses `curve.warmup.op.list` (FuseOpGetXattr) to differentiate with warmup query.